app/components/collections/form_component.html.erb

```diff
@@ -114,10 +114,7 @@
   <section>
     <header>Collection participants</header>
     <div class="mb-3 row">
-      <p class="col-sm-2 col-form-label">Created by</p>
-      <div class="col-sm-10 col-xl-8">
-        <%= collection_form.creator %>
```

Comment:
Can this method be removed from the form now?

Reply:
yes good catch, i think so, i removed the property from the form and pushed a new commit